pipelines/_ingredients/.ipynb_checkpoints/cpDeferredCharge-checkpoint.yaml

```diff
@@ -0,0 +1,32 @@
+description: cp_pipe CTI calibration construction
```

Comment:
I'm not sure about this checkpoint file. I don't think we want it here and should be deleted. Why did you add it?

Reply:
This was a mistake. This file was added because I edited my branch via JupyterLab on USDF. I'll remove it and work in vim from now on.